immix/llvm/memory_manager.cpp

```diff
@@ -80,12 +82,16 @@ static void roundTripDestroy(void *object)
 {
   delete static_cast<SectionMemoryManager *>(object);
 }
+
+
 extern "C"
 {
-  void CreatePLJITEngine(LLVMExecutionEngineRef *jit, LLVMModuleRef module, unsigned int opt, stackmap_cb cb)
+  LLVMExecutionEngineRef * CreatePLJITEngine( LLVMModuleRef module, unsigned int opt, stackmap_cb cb)
   {
+    LLVMExecutionEngineRef *jit = new LLVMExecutionEngineRef();
```

Comment:
I'm curious when do you free this pointer as you `new` manually. Can consider to use `shared_ptr` or `unique_ptr.

Reply:
in fact, LLVMExecutionEngineRef itself is a pointer, so there's no need to initialize it. I'll remove new expression later